crates/bitwarden-threading/src/thread_bound_runner.rs

```diff
@@ -0,0 +1,227 @@
+#![allow(dead_code)]
+#![allow(unused_variables)]
+
+use std::{future::Future, pin::Pin, rc::Rc};
+
+use bitwarden_error::bitwarden_error;
+use thiserror::Error;
+#[cfg(not(target_arch = "wasm32"))]
+use tokio::task::spawn_local;
+#[cfg(target_arch = "wasm32")]
+use wasm_bindgen_futures::spawn_local;
```

Comment:
The `wasm_bindgen_futures` crate is only in the dev-dependencies, so this will fail to build.